### DISCUSSION TITLE
Is there a version for pc if there is where can I get it?

### DISCUSSION THREAD

**Discussion by DiamondElement** [thatskymod/Sky-CotL-Scripts#102] [Q&A] [answered]
I need to know I really want to use canvas for steam pc.

**Comment by thatskymod** [comment, marked as answer]
Hello,

I believe we can consider this case resolved given that we now have a stable, well-maintained customization for the PC/Steam version. With the new [installer app](https://github.com/XeTrinityz/ThatSkyApp/releases), downloading and deploying the mod should be straightforward.

Credits to @XeTrinityz for making this possible.

Thank you for your contribution!
